tun2socks/tun2socks.c

```diff
@@ -297,9 +301,9 @@ LinkedList1 tcp_clients;
 // number of clients
 int num_clients;
 
-#ifdef __ANDROID__
 // Address of dnsgw
 BAddr dnsgw;
+#ifdef __ANDROID__
```

Comment:
Why move this?

Reply:
in this line, `dnsgw` is used. 
https://github.com/shadowsocks/badvpn/blob/fb01eb983915c9a09a690ad44c6028dd87500ec7/tun2socks/tun2socks.c#L633